utils/data.py

```
@@ -0,0 +1,46 @@
+# utils/data.py
+""" JSON 파일을 임시 DB처럼 다루기 위한 최소 유틸 """
+
+import json
+from pathlib import Path
+from typing import Any
+
+from fastapi import HTTPException, status
+
+
+def read_json(path: Path) -> Any:
```

Ruff (0.14.11)

Dynamically typed expressions (typing.Any) are disallowed in `read_json`

(ANN401)